daemon/create_test.go

```diff
@@ -1,21 +0,0 @@
-package daemon // import "github.com/docker/docker/daemon"
-
-import (
-	"testing"
-
-	"github.com/docker/docker/api/types/network"
-	"gotest.tools/v3/assert"
-	is "gotest.tools/v3/assert/cmp"
-)
-
-// Test case for 35752
-func TestVerifyNetworkingConfig(t *testing.T) {
```

Comment:
I'd prefer not to delete this test, and I'd even prefer to expand it to test interesting cases. But given that `verifyNetworkingConfig` is now a method of `daemon`, it's quite hard. I'd have to instantiate a daemon with a working libnetwork controller.

To make it easier to test, I thought about keeping `verifyNetworkingConfig` as a function and add a new `networkFinder func(string) *libnetwork.Network` parameter to it. It'd take `daemon.FindNetwork` in the daemon code, and a mock in this test. However, I'm not sure if that's a common / acceptable pattern.